README.md

```diff
@@ -1 +1,189 @@
+- const CLIENT_SECRET = 'your client secret';
+```
+
+## Start React development server
+npm start
+
+## Jest
+> To run testing
+```
+npm test
+```
+## Linters
+>To run linters
+```
+npx eslint "**/*.{js,jsx}
+npx stylelint "**/*.{css,scss}
+```
+
+## Backend pull request
+
+You can find the link to the frontend pull request [here](https://github.com/skabeo/backend_capstone/pull/35)
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a id="authors"></a>
+
+👤 **Spencer Okyere**
+
+- GitHub: [@skabeo](https://github.com/skabeo)
+- Twitter: [@black_okyere](https://twitter.com/black_okyere)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/spencer-okyere)
+
+👤 **Abdel-Khafid SALAOU**
+
+- GitHub: [@githubhandle](https://github.com/adeola003)
+- Twitter: [@twitterhandle](https://twitter.com/khadaf6)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abdel-khafid-salaou)
+
+👤 **Brian Kimutai Yegon**
+
+- GitHub: [@yegonkimutai](https://github.com/yegonkimutai)
+- Twitter: [@yegonbrian8](https://twitter.com/yegonbrian8)
+- LinkedIn: [Brian Yegon](https://www.linkedin.com/in/brian-yegon-0717a1241/)
+
+
+👤 **Juan Diaz**
+
+- GitHub: [@juanmanuelbyc](https://github.com/juanmanuelbyc)
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Only admin can add and delete properties**
+- **User will need to confirm an email before logging in**
+- **Users will get confirmation emails after reserving a property**
+- **Provide more information in properties details page**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a id="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/skabeo/frontend-capstone/issues).
+
+If you would like to contribute to this project, you can follow these steps:
+
+1. Fork the repository.
+2. Create a new branch for your changes.
+3. Make your changes and commit them.
+4. Push your changes to your forked repository.
+5. Create a pull request back to the original repository.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this repo dont forget to give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse community and the partners.
+I also give thanks to [Murat Korkmaz on Behance](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005579?log_shim_removal=1), the author of original design we got inspiration from.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/skabeo/frontend-capstone/blob/develop/LICENSE) licensed.
```

Comment:
- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

The file name is `LICENSE.md`, not `LICENSE`.

Kindly fix the broken link.

_note:  The same issue was present on the backend readme._


That way, your app will be well documented 💪